### ISSUE TITLE
[!] Invalid `Podfile` file: cannot load such file -- /../.symlinks/plugins/awesome_notifications/ios/Scripts/AwesomePodFile.

### ISSUE THREAD

**Issue by VarPDev** [rafaelsetragni/awesome_notifications#936]
After inserting the necessary changes needed on the podfile, I keep getting the following error when trying to install pods:

with command `pod install`

```
pod install --repo-update --project-directory=ios


[!] Invalid `Podfile` file: cannot load such file -- /Users/pasquale.delucia/Apps/scuolazooviaggi_app/.symlinks/plugins/awesome_notifications/ios/Scripts/AwesomePodFile.

 #  from /Users/pasquale.delucia/Apps/scuolazooviaggi_app/ios/Podfile:55
 #  -------------------------------------------
 #  awesome_pod_file = File.expand_path(File.join('plugins', 'awesome_notifications', 'ios', 'Scripts', 'AwesomePodFile'), '.symlinks')
 >  require awesome_pod_file
 #  update_awesome_main_target_settings('Runner', File.dirname(File.realpath(__FILE__)), flutter_root)
 #  -------------------------------------------
```

with command `flutter build ios`

```
Building com.scuolazoo.travel for device (ios-release)...
Automatically signing iOS for device deployment using specified development team in Xcode
project: 37CLX8358U
Running pod install...                                             926ms
CocoaPods' output:
↳

    [!] Invalid `Podfile` file: cannot load such file --
    /Users/pasquale.delucia/Apps/scuolazooviaggi_app/ios/.symlinks/plugins/awesome_notifications/
    ios/Scripts/AwesomePodFile.

     #  from /Users/pasquale.delucia/Apps/scuolazooviaggi_app/ios/Podfile:55
     #  -------------------------------------------
     #  awesome_pod_file = File.expand_path(File.join('plugins', 'awesome_notifications', 'ios',
     'Scripts', 'AwesomePodFile'), '.symlinks')
     >  require awesome_pod_file
     #  update_awesome_main_target_settings('Runner', File.dirname(File.realpath(__FILE__)),
     flutter_root)
     #  -------------------------------------------

    /Library/Ruby/Gems/2.6.0/gems/cocoapods-core-1.15.2/lib/cocoapods-core/podfile.rb:335:in
    `rescue in block in from_ruby'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-core-1.15.2/lib/cocoapods-core/podfile.rb:329:in
    `block in from_ruby'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-core-1.15.2/lib/cocoapods-core/podfile.rb:50:in
    `instance_eval'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-core-1.15.2/lib/cocoapods-core/podfile.rb:50:in
    `initialize'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-core-1.15.2/lib/cocoapods-core/podfile.rb:327:in
    `new'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-core-1.15.2/lib/cocoapods-core/podfile.rb:327:in
    `from_ruby'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-core-1.15.2/lib/cocoapods-core/podfile.rb:293:in
    `from_file'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.15.2/lib/cocoapods/config.rb:206:in `podfile'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.15.2/lib/cocoapods/command.rb:160:in
    `verify_podfile_exists!'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.15.2/lib/cocoapods/command/install.rb:46:in `run'
    /Library/Ruby/Gems/2.6.0/gems/claide-1.1.0/lib/claide/command.rb:334:in `run'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.15.2/lib/cocoapods/command.rb:52:in `run'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.15.2/bin/pod:55:in `<top (required)>'
    /usr/local/bin/pod:23:in `load'
    /usr/local/bin/pod:23:in `<main>'

Error running pod install
```

this is the part of my pod file I modified for awesome notification

```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    flutter_additional_ios_build_settings(target)
  end
    ################  Awesome Notifications pod modification 1 ###################
    awesome_pod_file = File.expand_path(File.join('plugins', 'awesome_notifications', 'ios', 'Scripts', 'AwesomePodFile'), '.symlinks')
    require awesome_pod_file
    update_awesome_pod_build_settings(installer)
    ################  Awesome Notifications pod modification 1 ###################
end

################  Awesome Notifications pod modification 2 ###################
awesome_pod_file = File.expand_path(File.join('plugins', 'awesome_notifications', 'ios', 'Scripts', 'AwesomePodFile'), '.symlinks')
require awesome_pod_file
update_awesome_main_target_settings('Runner', File.dirname(File.realpath(__FILE__)), flutter_root)
################  Awesome Notifications pod modification 2 ###################
```

packages installed

```
  awesome_notifications_core: ^0.9.1 # <~ always ensure to use the latest version
  awesome_notifications: any # <~ this version is managed by awesome_notifications_core package
```

**Comment by hnvcam** [comment]
This is my workaround:

- pubspec.yaml:
`
awesome_notifications: ^0.9.2
`
don't include awesome_notifications_core which always resolves to awesome_notifications: 0.8.2
- Run `flutter pub get`
- If you have problem with `intl: 0.19.0`:
```
dependency_overrides:
  intl: ^0.19.0
```
- Remove the ${ProjectPath}/ios/.symlinks/plugins/awesome_notifications
- Run `pod install` again

**Comment by abdgul99** [comment]
I have the same issue
 `[!] Invalid `Podfile` file: cannot load such file -- /Users/master/Documents/Github/gmll/ios/.symlinks/plugins/awesome_notifications/ios/Scripts/AwesomePodFile.

 #  from /Users/master/Documents/Github/gmll/ios/Podfile:59
 #  -------------------------------------------
 #  awesome_pod_file = File.expand_path(File.join('plugins', 'awesome_notifications', 'ios', 'Scripts', 'AwesomePodFile'), '.symlinks')
 >  require awesome_pod_file
 #  update_awesome_main_target_settings('Runner', File.dirname(File.realpath(__FILE__)), flutter_root)`

**Comment by abdgul99** [comment]
> This is my workaround:
> 
> * pubspec.yaml:
>   `awesome_notifications: ^0.9.2`
>   don't include awesome_notifications_core which always resolves to awesome_notifications: 0.8.2
> * Run `flutter pub get`
> * If you have problem with `intl: 0.19.0`:
> 
> ```
> dependency_overrides:
>   intl: ^0.19.0
> ```
> 
> * Remove the ${ProjectPath}/ios/.symlinks/plugins/awesome_notifications
> * Run `pod install` again

solved

**Comment by AntoniPochopien** [comment]
i have same issue. Given fix is not working for me

`Invalid `Podfile` file: cannot load such file -- /my_project/ios/.symlinks/plugins/awesome_notifications/ios/Scripts/AwesomePodFile.

 #  from /my_project/ios/Podfile:54
 #  -------------------------------------------
 #  awesome_pod_file = File.expand_path(File.join('plugins', 'awesome_notifications', 'ios', 'Scripts', 'AwesomePodFile'), '.symlinks')
 >  require awesome_pod_file
 #  update_awesome_main_target_settings('Runner', File.dirname(File.realpath(__FILE__)), flutter_root)
 #  -------------------------------------------
`

here is part from my poubspec

` 
awesome_notifications: ^0.9.2

dependency_overrides:
  intl: ^0.19.0
`
(I removed the core package dependency, but I'm encountering the same error both with and without it. I've tried versions 9.2 and 9.1.)

`
[!] CocoaPods could not find compatible versions for pod "IosAwnCore":
  In snapshot (Podfile.lock):
    IosAwnCore (= 0.8.0, ~> 0.8.0)

  In Podfile:
    awesome_notifications (from `.symlinks/plugins/awesome_notifications/ios`) was resolved to 0.9.0, which depends on
      IosAwnCore (~> 0.9.1)
`

**Comment by ckt22** [comment]
> > This is my workaround:
> > 
> > * pubspec.yaml:
> >   `awesome_notifications: ^0.9.2`
> >   don't include awesome_notifications_core which always resolves to awesome_notifications: 0.8.2
> > * Run `flutter pub get`
> > * If you have problem with `intl: 0.19.0`:
> > 
> > ```
> > dependency_overrides:
> >   intl: ^0.19.0
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * Remove the ${ProjectPath}/ios/.symlinks/plugins/awesome_notifications
> > * Run `pod install` again
> 
> solved

Encountered same issue and this fix worked. Thanks.
